src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -158,9 +282,49 @@ const GroupCheckbox = styled(Box)`
   }
 `;
 
-const StyledCount = styled(Count)`
+const PrimaryCount = styled(Count)`
+  font-size: 18px;
+  color: ${p => p.theme.gray700};
+`;
+
+const SecondaryCount = styled(({value, ...p}) => (
+  <React.Fragment>
+    {'/'}
+    <Count {...p} value={value} />
+  </React.Fragment>
+))`
   font-size: 18px;
-  color: ${p => p.theme.gray600};
+  color: ${p => p.theme.gray500};
+`;
```

Comment:
I was thinking using `:before { content: '/' }`, but this works

Reply:
That is much cleaner, I'll do it that way